about.html

```diff
@@ -0,0 +1,104 @@
+                    </ul>
+                </div>
+                <div class="menu-bar">
+                    <div class="burger" id="btn">
+                        <span class="bar" id="btn1"></span>
+                        <span class="bar"></span>
+                        <span class="bar" id="btn2"></span>
+                    </div>
+                </div>
+            </nav>
+        </header>
+        
+        <div class="heading">
+            <p class="title">Making Differences in the life of others</p>
+            <h1>Leading the Way in Medical Excellence</h1>
+            <p class="description">
+                We’ve been named a Five-Star Hospital by the Centers for Medicare and Medicaid Services (CMS) for the quality of our patient care, and recognized as one of the nation’s top critical access hospitals by Becker’s Hospital Review. We’ve also earned The Joint Commission’s Gold Seal of Approval™.
+            </p>
+            <p class="check-in-text">Please contact us per Email for futher questions about Hudson</p>
+            <p class="second-title">
+                Hudson2023@hudson.org
+            </p>
+        </div>
+    </section>
+
+    <!--- main section--->
+    <section class="logo-summit">
+        <h3>HUDSON 2023 Logo</h3>
+        <p>
+            The logo of Hudson 2023 was decided through the logo competition from 8 June to 7 July.
+            <span class="none">
+                After online public voting and Selection Committee's judging process.the logo from Naresh Agrawal won the prize.
+            After consultation with Mr.Agrawal we optimized the prize-winning-logo for this years summit as following.
+            </span> 
+         </p>
+         <a class="logo-summit-2023" href="">HUDSON</a> 
+    </section>
+
+    <!--- The Past--->
+
+    <section class="history">
+        <h3>See the past HUDSON.</h3>
+        <p>Take a look at the last two Hudson summit which took place in Buenos and Warsaw</p>
+        <img src="./images/base/june.svg" alt="">
+        <img src="./images/base/july.svg" alt="">
+    </section>
+
+     <!--- Prtners --->
+
+     <section class="partners">
+        <h1>Partner</h1>
+        <div class="partners-container">
+            <div>
+                <img src="./images/sponsors/bray.png" alt="">
+            </div>
+            <div>
+                <img src="./images/sponsors/bray.png" alt="">
+            </div>
+            <div>
+                <img src="./images/sponsors/bray.png" alt="">
+            </div>
+        </div>
+    </section>
+
+    <footer>
+        <div class="copy-right">
+            <p class="copy-right-author">2023 HUDSON Project. Made by Brighton Mtandi.</p>
+            <p>All rights reserved.</p>
+            <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Molestias voluptatum minima ipsam earum tenetur hic quis quasi suscipit aliquam corrupti.</p> 
+        </div>
+    </footer>

```

Comment:
- [ ] [REQUIRED] Good job so far! Please `About` page's footer should be `dark mode` and `Partner` section should not displayed also pay attention to fonts for `headings` 🙏🏼 

![image](https://github.com/brighton0102/HTML-CSS-JavaScript-Capstone-Project/assets/63915024/22f109af-9b83-4a70-9a4d-2e91f6835757)